mmcv/ops/csrc/common/cuda/voxel_query_vector_pool.cuh

```diff
@@ -0,0 +1,189 @@
+#ifndef VOXEL_QUERY_VECTOR_POOL_CUH
+#define VOXEL_QUERY_VECTOR_POOL_CUH
+
+#ifdef MMCV_USE_PARROTS
+#include "parrots_cuda_helper.hpp"
+#else
+#include "pytorch_cuda_helper.hpp"
+#endif
+
+
+__global__ void stack_vector_pool_cuda_kernel(
```

Comment:
Is this kernel write by yourself or modify from other repo?
If it comes from other repo, please add the comment about the link.